sdk/examples/how_tos/nfts/send_nft.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will send an nft.
+//! Rename `.env.example` to `.env` first.
+//!
+//! `cargo run --example send_nft --release`
+
+use iota_sdk::{
+    types::block::address::Bech32Address,
+    wallet::{Result, SendNftParams, Wallet},
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    // Create the wallet
+    let wallet = Wallet::builder().finish().await?;
+
+    // Get the account we generated with `01_create_wallet`
+    let account = wallet.get_account("Alice").await?;
+    // May want to ensure the account is synced before sending a transaction.
+    let balance = account.sync(None).await?;
+
+    // Get the first nft
+    if let Some(nft_id) = balance.nfts().first() {
+        // Set the stronghold password
+        wallet
+            .set_stronghold_password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
+            .await?;
+
+        let outputs = vec![SendNftParams {
+            address: Bech32Address::try_from_str("rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu")?,
+            nft_id: *nft_id,
+        }];
+
+        let transaction = account.send_nft(outputs, None).await?;
+        println!("Transaction sent: {}", transaction.transaction_id);
+
+        // Wait for transaction to get included
+        let block_id = account
+            .retry_transaction_until_included(&transaction.transaction_id, None, None)
+            .await?;
+
+        println!(
+            "Block included: {}/block/{}",
+            std::env::var("EXPLORER_URL").unwrap(),
+            block_id
+        );
```

Comment:
Different output as in the `## Expected Output`

Reply:
Yeah. Noted the output for nodejs. 
Need to work on this. Either make them all same output or add diff outputs for each in tabs.